package-lock.json

```diff
@@ -13916,16 +13855,6 @@
       "integrity": "sha1-AerA/jta9xoqbAL+q7jB/vfgDqs=",
       "dev": true
     },
-    "postcss": {
```

Comment:
This line is the payoff of this entire thing - we need this dependency to go away!  It's preventing us from upgrading to webpack 5, since the latest and greatest packages in the ecosystem all require PostCSS 8, and it gets installed as a peer dependency.  Dependencies like this line were superceding our attempts at installing PostCSS 8 and breaking everything.